www/src/components/guidelines/color/card.js

```diff
@@ -0,0 +1,96 @@
+import React from "react"
+import MdLaunch from "react-icons/lib/md/launch"
+
+import { Box, Flex, Link, Text } from "../system"
+import BoxWithBorder from "../box-with-border"
+import { getTextColor } from "../../../utils/guidelines/color"
+import { colors } from "../../../utils/presets"
+
+const ColorValue = ({ label, inverted, value, href }) => (
+  <Box mt={4} px={2} css={{ flexShrink: 0, flexBase: `50%` }}>
+    <Text
+      color={inverted ? `whiteFade.70` : `blackFade.70`}
+      fontFamily="header"
+      fontSize={1}
+    >
+      {label}
+    </Text>
+    <Flex alignItems="center">
+      <Text
+        fontSize={1}
+        css={{ whiteSpace: `nowrap` }}
+        color={inverted ? `whiteFade.80` : `blackFade.80`}
+      >
+        {value}
+        {href && (
+          <>
+            {` `}
+            <Link
```

Comment:
The PMS link is missing an accessible name. Perhaps some visually-hidden text could be added, or an `aria-label` indicating where the link will go?

The element returned from axe-core:
``` <a href="https://www.pantone.com/color-finder/2077-C" class="css-18d996j-Box-Link-ColorValue e14qwfqd0">```

Reply:
Added a screenreader-only bit of copy "Pantone Matching System reference" in 20ce300.